backend/battle/tests/test_api.py

```diff
@@ -0,0 +1,354 @@
+from unittest.mock import patch
+from model_bakery import baker
+from rest_framework.test import APIClient, APITestCase
+from django.urls import reverse
+from django.conf import settings
+from battle.models import Battle, Team
+from battle.serializers import BattleSerializer
+
+
+class APITestCaseUtils(APITestCase):
+    def setUp(self):
+        self.user_1 = baker.make("users.User")
+        self.opponent = baker.make("users.User")
+
+        self.auth_client = APIClient()
+        self.auth_client.force_authenticate(user=self.user_1)
+
+
+class ListBattleEndpointTest(APITestCaseUtils):
+
+    def test_login_user_can_acess_battle_list(self):
+        response = self.auth_client.get(reverse('battle-list'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_endpoint_returns_empty_list(self):
+        battle = baker.make('battle.Battle', creator=self.user_1)
+        response = self.auth_client.get(reverse('battle-list'))
+        matching_battles = BattleSerializer([battle], many=True)
+        self.assertEqual(response.status_code, 200)
+        self.assertCountEqual(matching_battles.data, response.json())
```

Comment:
Shouldn't this be an empty list?

Reply:
yup! I didn't realize that I was creating a battle, sorry :)